src/warnings.h

```diff
@@ -6,18 +6,13 @@
 #ifndef BITCOIN_WARNINGS_H
 #define BITCOIN_WARNINGS_H
 
-#include <string>
+#include <vector>
 
 struct bilingual_str;
 
 void SetMiscWarning(const bilingual_str& warning);
 void SetfLargeWorkInvalidChainFound(bool flag);
-/** Format a string that describes several potential problems detected by the core.
- * @param[in] verbose bool
- * - if true, get all warnings separated by <hr />
- * - if false, get the most important warning
- * @returns the warning string
- */
-bilingual_str GetWarnings(bool verbose);
+/** Return potential problems detected by the node. */
```

Comment:
nit:  Looks like the old function comment header used Doxygen syntax.  Would recommend keeping this style, but feel free to disregard this nit since the return type seems straightforward, and the function is small/digestible.

```diff
diff --git a/src/warnings.h b/src/warnings.h
index 0b8eb2c9a1..4ae05d4862 100644
--- a/src/warnings.h
+++ b/src/warnings.h
@@ -12,7 +12,10 @@ struct bilingual_str;
 
 void SetMiscWarning(const bilingual_str& warning);
 void SetfLargeWorkInvalidChainFound(bool flag);
-/** Return potential problems detected by the node. */
+/** Return potential problems detected by the node.
+ *
+ * @returns each warning string as a vector element.
+ */
 std::vector<bilingual_str> GetWarnings();
 
 #endif // BITCOIN_WARNINGS_H

```

Reply:
I generally tend to prefer to doxygen-ify documentation, but in this case I'm not sure the extra line adds anything over the return type and function name, so then is it just unnecessary boilerplate that makes the documentation less instead of more clear? No strong view either way, so will leave this open.